### PR TITLE
Fix Amberscript example workflows

### PR DESCRIPTION
**Pull request by lkiesow**
This patch fixes the amberscript example workflows in the documentation which were missing some mandatory configuration options (like the target flavor in the attach operation).

### Your pull request should…

* [x] have a concise title
* [x] [close an accompanying issue](https://docs.opencast.org/develop/developer/#participate/development-process/#automatically-closing-issues-when-a-pr-is-merged) if one exists
* [x] [be against the correct branch](https://docs.opencast.org/develop/developer/development-process#acceptance-criteria-for-patches-in-different-versions)
* [x] include migration scripts and documentation, if appropriate
* [x] pass automated tests
* [x] have a clean commit history
* [x] [have proper commit messages (title and body) for all commits](https://medium.com/@steveamaza/e028865e5791)

### CODE REVIEW

File: docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md [hunk 1]
```diff
@@ -40,85 +40,48 @@ dropdowns below.
 
 <summary>amberscript-attach-transcription.xml</summary>
```
Comment (gregorydlogan) [inline docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md]:
```suggestion
<summary>amberscript-attach-transcription.yml</summary>
```

File: docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md [hunk 2]
```diff
@@ -127,59 +90,37 @@ dropdowns below.
 
 <summary>amberscript-start-transcription.xml</summary>
```
Comment (gregorydlogan) [inline docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md]:
```suggestion
<summary>amberscript-start-transcription.yml</summary>
```

**Review by gregorydlogan — APPROVED** [review]
There's no documentation in either WOH page regarding which parameters are required, so I'm assuming this is right :sweat_smile:.  Minor complaints because you're going from xml to yml, otherwise LGTM.

File: docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md (hunk 2 above)
Comment (lkiesow) [reply docs/guides/admin/docs/configuration/transcription.configuration/amberscripttranscripts.md]: Good catch. Updated.

**Comment by lkiesow** [comment]
Fixed the `xml` → `yaml` in line 130 as well

**Pull request merged**